### DISCUSSION TITLE
Rectangle like commands accessable through the menu bar

### DISCUSSION THREAD

**Discussion by alper** [nikitabobko/AeroSpace#1010] [feature-ideas]
I think it could help learnability of the tool to have some common (or configurable!) actions that you can trigger directly from the menubar.

Rectangle does this and it's very helpful in the beginning even if you don't use it that much later anymore.

(Also I'm in and I can at some point look into contributing some of these things.)

**Comment by emgeee** [comment]
+1 to this idea. With a lot of productivity tools I find that the majority of common tasks I can pick up hotkeys for relatively quickly but for certain long-tail functions it isn't worth binding/memorizing the hotkey. Being able to use those features through a drop-down menu, while not as efficient as using a hotkey, is often fast enough for some little used (but still used!) features.
